packages/ember-runtime/lib/mixins/mutable_array.js

```diff
@@ -19,6 +19,15 @@ import MutableEnumerable from './mutable_enumerable';
 import Enumerable from './enumerable';
 import { Error as EmberError } from 'ember-debug';
 
+function replace(array, idx, amt, objects) {
```

Comment:
I would have expected this to be much closer to [NativeArray#replace](https://github.com/emberjs/ember.js/blob/b3ad8b4999cb1043b1f76debf187187c139327cb/packages/ember-runtime/lib/system/native_array.js#L54-L71)...